src/libbson/src/bson/bson.c

```diff
@@ -2260,6 +2282,34 @@ bson_copy_to_excluding_noinit_va (const bson_t *src,
 }
 
 
+void
+bson_copy_to_including_noinit_va (const bson_t *src,
```

Comment:
Suggest moving these to `mongoc-util.c` and renaming, since they are declared within `mongoc-util-private.h`